dokka-integration-tests/gradle/src/main/kotlin/org/jetbrains/dokka/it/gradle/TestConstants.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2014-2025 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package org.jetbrains.dokka.it.gradle
+
+object TestConstants {
+    val DokkaHtmlAssetsFiles = listOf(
```

Comment:
```suggestion
object TestConstants {
    /**
     * Dokka HTML files that are only used for styling the generated HTML content.
     *
     * These files should be excluded from file-content assertion checks, to avoid
     * minor style changes affecting the tests.
     */
    val DokkaHtmlAssetsFiles = listOf(
```

suggestion: add a small comment to explain. I would find it useful. Feel free to reword it.